students/tony7kh/hw-8/task-3.js

```diff
@@ -0,0 +1,12 @@
+function searchSubstringIndex(initialString, searchingSubstring) {
```

Comment:
> Function should return an amount of the searching string in the initial one
or an exception string 'There is not 'cat' in the initial string.' if is not such substring.

Reply:
Also this code looks pretty the same as students/AntonBakaiev/hw-8/task-3.js 🤔